python/sglang/srt/model_executor/cuda_graph_runner.py

```diff
@@ -263,11 +288,20 @@ def __init__(self, model_runner: ModelRunner):
         self.attn_tp_size = get_attention_tp_size()
         self.attn_tp_rank = get_attention_tp_rank()
 
+        self.max_num_token = 0
```

Comment:
hmm it seems the field is not used globally, thus what about removing it and only have some local variables